docs/interacting/web-ui.md

```diff
@@ -0,0 +1,35 @@
+# Web application
+
+The web application is hosted at [skywalking-banyandb-webapp](http://localhost:17913/) when you boot up the BanyanDB server.
+The web ui could use to query data stored in streams, measures, and properties.
+
+## Stream
```

Comment:
Add, edit, and delete a specific stream on this page. Please add relevant operations here.

Reply:
You can also view and manage the index-rule and index-rule-binding in a stream group in this page.